test/autorest/generated/lrogroup/pollers.go

```diff
@@ -107,6 +117,9 @@ func (p *productArrayPoller) Poll(ctx context.Context) (*http.Response, error) {
 }
 
 func (p *productArrayPoller) FinalResponse(ctx context.Context) (*ProductArrayResponse, error) {
+	if p.Done() && p.pollerMethodVerb() == "PUT" && p.resp != nil && p.resp.ProductArray != nil {
```

Comment:
Can use `http.MethodPut` instead of `"PUT"`.

Reply:
Done